doc/srcdeps.yaml

```diff
@@ -183,6 +188,62 @@ maven:
     - my-bare-breaker-property
     - my-breaker-property=my-breaker-value
 
+  # `failWithout` - a configuration node to define conditions whose absence will make the outer Maven build fail
```

Comment:
> Does it make more sense to have a "negate" failWith instead? For instance, failWith=!srcdeps.enabled ?

I worked on that idea till yesterday. there is also a PR https://github.com/srcdeps/srcdeps-core/pull/11 (Are you asking because you have seen it? :) I abandoned it because I found the semantics not intuitive enough. E.g. if you have multiple negated props combined with non-negated ones, one has to eval the negated differently and separately from the non-negated ones. I found it hard to formulate reasonably simple and sufficiently informative docs and messages for the user when something went wrong. Having the negative assertions separated from the non-negated ones directly in the config (like in the present proposal) mirrors better what the code does and is easier to explain and understand IMO.